tests/unit/sftp_test.py

```diff
@@ -18,11 +20,21 @@ def sftp_session(ssh_client_session):
         del sftp_sess  # noqa: WPS420
 
 
-@pytest.fixture
-def transmit_payload():
-    """Generate a binary test payload."""
-    uuid_name = uuid.uuid4()
-    return 'Hello, {name!s}'.format(name=uuid_name).encode()
+@pytest.fixture(
+    params=(32, 1024 + 1),
+    ids=('small-payload', 'large-payload'),
+)
+def transmit_payload(request: pytest.FixtureRequest):
+    """Generate binary test payloads of assorted sizes.
+
+    The choice 32 is arbitrary small value.
+
+    The choice 1024 + 1 is meant to be 1B larger than the chunk size used in
+    :file:`sftp.pyx` to make sure we excercise at least two rounds of reading/writing.
+    """
+    payload_len = request.param
+    random_bytes = [ord(random.choice(string.printable)) for _ in range(payload_len)]  # NOSONAR
```

Comment:
It's unclear what this is ignoring. Let's not add suppressions w/o justification in a comment.

Reply:
Let's revert it, then, and I'll make sonar to shut up on the web UI.